tools/dockerfile/test/android_ndk/Dockerfile

```diff
@@ -93,15 +93,27 @@ RUN apt-get update && apt-get -y install openjdk-8-jdk && apt-get clean
 
 # Install Android SDK
 ENV ANDROID_SDK_VERSION 4333796
-RUN mkdir -p /opt/android-sdk && cd /opt/android-sdk &&     wget -q https://dl.google.com/android/repository/sdk-tools-linux-$ANDROID_SDK_VERSION.zip &&     unzip -q sdk-tools-linux-$ANDROID_SDK_VERSION.zip &&     rm sdk-tools-linux-$ANDROID_SDK_VERSION.zip
+RUN mkdir -p /opt/android-sdk && cd /opt/android-sdk && \
+    wget -q https://dl.google.com/android/repository/sdk-tools-linux-$ANDROID_SDK_VERSION.zip && \
+    unzip -q sdk-tools-linux-$ANDROID_SDK_VERSION.zip && \
+    rm sdk-tools-linux-$ANDROID_SDK_VERSION.zip
 ENV ANDROID_SDK_PATH /opt/android-sdk
-
-# Install Android NDK and cmake using sdkmanager
-RUN mkdir -p ~/.android && touch ~/.android/repositories.cfg
+ENV ANDROID_HOME /opt/android-sdk
 RUN yes | $ANDROID_SDK_PATH/tools/bin/sdkmanager --licenses  # accept all licenses
-RUN $ANDROID_SDK_PATH/tools/bin/sdkmanager ndk-bundle 'cmake;3.6.4111459'
-ENV ANDROID_NDK_PATH $ANDROID_SDK_PATH/ndk-bundle
-ENV ANDROID_SDK_CMAKE $ANDROID_SDK_PATH/cmake/3.6.4111459/bin/cmake
+
+# Install Android NDK
+RUN curl -L -o android-ndk.zip http://dl.google.com/android/repository/android-ndk-r16b-linux-x86_64.zip && \
```

Comment:
I am not sure which NDK version should be used. From my recent experiments, NDK 16 can make my code work, but some later versions cannot.

Reply:
AFAICT, this version is what was used in the original kokoro ubuntu16.04 workers (which is where the grpc_android job is currently running on). So starting with the same version makes sense (and we can adjust in the future if needed).